diff --git a/node_modules/@lekoarts/gatsby-theme-minimal-blog/gatsby-config.js b/node_modules/@lekoarts/gatsby-theme-minimal-blog/gatsby-config.js
index 5c5ba0f..9747c6a 100755
--- a/node_modules/@lekoarts/gatsby-theme-minimal-blog/gatsby-config.js
+++ b/node_modules/@lekoarts/gatsby-theme-minimal-blog/gatsby-config.js
@@ -38,7 +38,7 @@ module.exports = options => {
       },
       {
         resolve: `gatsby-plugin-feed`,
-        options: newsletterFeed,
+        options: newsletterFeed(options),
       },
       `gatsby-plugin-react-helmet`,
       `gatsby-plugin-typescript`,
diff --git a/node_modules/@lekoarts/gatsby-theme-minimal-blog/src/utils/newsletterFeed.js b/node_modules/@lekoarts/gatsby-theme-minimal-blog/src/utils/newsletterFeed.js
index d83abf9..36d7a33 100755
--- a/node_modules/@lekoarts/gatsby-theme-minimal-blog/src/utils/newsletterFeed.js
+++ b/node_modules/@lekoarts/gatsby-theme-minimal-blog/src/utils/newsletterFeed.js
@@ -1,6 +1,6 @@
 /* eslint arrow-body-style: 0 */
 
-module.exports = {
+module.exports = ({ feed: feedOptions }) => ({
   query: `
     {
       site {
@@ -41,6 +41,7 @@ module.exports = {
       `,
       output: `rss.xml`,
       title: `Minimal Blog - @lekoarts/gatsby-theme-minimal-blog`,
+      ...feedOptions
     },
   ],
-}
+})
